scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/goals/Goal.java

```diff
@@ -32,6 +32,12 @@ public class Goal {
    * state constraints applying to the goal
    */
   protected Expression<Windows> resourceConstraints;
+  
+  /**
+   * Whether to resimulate after this goal or use stale results
```

Comment:
Minor, but this comment feels slightly inaccurate, since it's not whether _this_ goal uses stale results, but whether the _next_ goal uses stale results.